docs/specification/draft/basic/index.mdx

```diff
@@ -147,3 +147,35 @@ may reserve particular names for purpose-specific metadata, as declared in those
 
 - Unless empty, MUST begin and end with an alphanumeric character (`[a-z0-9A-Z]`).
 - MAY contain hyphens (`-`), underscores (`_`), dots (`.`), and alphanumerics in between.
+
+#### `icons`
+
+The `icons` property provides a standardized way for servers to expose visual identifiers for their resources, tools, prompts, and implementations. Icons enhance user interfaces by providing visual context and improving the discoverability of available functionality.
+
+Icons are represented as an array of `Icon` objects, where each icon includes:
+
+- `src`: A URI pointing to the icon resource (required). This can be:
+  - An HTTP/HTTPS URL pointing to an image file
+  - A data URI with base64-encoded image data
+- `mimeType`: Optional MIME type (e.g., "image/png", "image/svg+xml") if the server's type is missing or generic
```

Comment:
Could having an open-ended "mimeType" be dangerous? What happens if a server declares that its icon's mimeType is "text/javascript"? Should we replace mimeType with "imageType", which only has a few options for whatever follows "image/" in the mimeType of the HTTP response?